### ISSUE TITLE
No Color in MOTD

### ISSUE THREAD

**Issue by mrunningmasterc** [strumswell/ServerlistMOTD-X#12]
Hi, so I'm having a problem with color in the motd as you can see from the pictures. I'm using the latest 1.15 version on a paper 1.16. The hover portion works correctly with color but not the actual motd. I also saw your other versions for 1.16 with protocol support (used the 1.16 dev builds) and without but neither of them fixed the issue.

![Screen Shot 2020-07-30 at 1 17 35 PM](https://user-images.githubusercontent.com/63335629/88953536-15f40700-d267-11ea-83b4-836f1fdcc96f.png)
![Screen Shot 2020-07-30 at 1 17 44 PM](https://user-images.githubusercontent.com/63335629/88953544-17bdca80-d267-11ea-90eb-7801df70e563.png)
![Screen Shot 2020-07-30 at 1 18 41 PM](https://user-images.githubusercontent.com/63335629/88953637-358b2f80-d267-11ea-9037-2ad416de9559.png)

**Comment by strumswell** [comment]
Hey there. 👋 I‘m going to look into it! Can you send me your config.yml as a pastebin.com link?

**Comment by mrunningmasterc** [comment]
I have no Idea, as the hover text works fine with the previous color code system.

https://pastebin.com/tBak1a2U

**Comment by strumswell** [comment]
I am going to look into it tomorrow. Can you try removing the stars from the random motds? Are the colors coming back after that? Just a guess.

**Comment by mrunningmasterc** [comment]
I can try it, it shouldn't be that though because the whitelist mode and the ban mode for motd doesn't even show color either. Theres no special characters or unicode in that either.

**Comment by strumswell** [comment]
I used the config.yml you provided and everything works as intended:

<img width="966" alt="Bildschirmfoto 2020-07-30 um 23 01 37" src="https://user-images.githubusercontent.com/6720421/88974404-dc4bec00-d2b8-11ea-8cfe-65cc7289e2af.png">

I used:
- Spigot 1.16.1-R0.1-SNAPSHOT
- ServerlistMOTD vX-2020-03-02
- ProtocolLib v4.5.1

Unicode chars were not a problem as you said. So I guess it's not a plugin specific problem.

Have you tried only running ServerlistMOTD and ProtocolLib on your server? Which version (exact) of Paper are you running?
